modules/manual-configuration-image-registry-qcr.adoc

```diff
@@ -5,21 +5,35 @@
 [id="manual-configuration-image-registry-qcr_{context}"]
 = Manually configuring Quay Container Registry
 
-You can integrate {product-title} with Quay Container Registry.
+You can integrate {product-title} ({product-title-short}) with Quay Container Registry. You can integrate with Quay using the following methods:
+
+- Integrating with the Quay public repository (registry): This method does not require authentication.
+- Integrating with a Quay private registry using a robot account: This method requires that you create a robot account to use with Quay (recommended). See the link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/use_red_hat_quay/use-quay-manage-repo#allow-robot-access-user-repo[Quay documentation] for more information.
```

Comment:
Aha, this is a subtle distinction in terminology which I missed for public versus private.

For the next release, I will replace **Public repositories** with **Public registry** in the info note item of the Quay integration form.

Reply:
It's really confusing but Quay seems to refer to their registry (which includes all public image repos) and repo (individual buckets in the registry, public or private) somewhat interchangeably. I mostly went from this page which seems to imply that the overall service is a registry, but I could be misunderstanding it. https://access.redhat.com/articles/quayio-help